package.json

```diff
@@ -2,7 +2,6 @@
   "name": "finesse-frontend",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://ai-cfia.github.io/finesse-frontend",
```

Comment:
why remove the homepage?

Reply:
think it was because of the multiple URLs